class.FlipPage.php

```diff
@@ -397,11 +401,11 @@ protected function addAllLinks()
             $this->addLink('Logout', $this->logoutUrl);
         }
         $aboutMenu = array(
```

Comment:
If we are going to do it this way. We probably should have the whole about menu loaded from settings to allow for unique contents.